iocore/net/SSLCertLookup.cc

```diff
@@ -45,26 +47,19 @@
 #endif /* SSL_CTX_set_tlsext_ticket_key_cb */
 
 struct SSLAddressLookupKey {
-  explicit SSLAddressLookupKey(const IpEndpoint &ip) : sep(0)
+  explicit SSLAddressLookupKey(const IpEndpoint &ip)
   {
-    static const char hextab[16] = {'0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'a', 'b', 'c', 'd', 'e', 'f'};
-
-    int nbytes;
-    uint16_t port = ntohs(ip.port());
-
-    // For IP addresses, the cache key is the hex address with the port concatenated. This makes the lookup
-    // insensitive to address formatting and also allow the longest match semantic to produce different matches
-    // if there is a certificate on the port.
-    nbytes = ats_ip_to_hex(&ip.sa, key, sizeof(key));
```

Comment:
This function dies at some point presumably?

Reply:
Yes.